src/react/webgl-renderer.js

```diff
@@ -131,9 +131,52 @@ export default class WebGLRenderer extends React.Component {
     }
   }
 
+  // Gets current size of canvas in css (logical/window) coordinates
+  _getCSSSize(canvas) {
```

Comment:
All these added methods are copy pasted from luma, we can import from `luma` once we finalize the fix.